app/models/miq_ae_domain.rb

```diff
@@ -100,6 +100,17 @@ def editable_property?(property)
 
   alias editable_contents? editable?
 
+  def update_attributes_api(hash)
```

Comment:
@lfu @mkanoor Do we know where the API is calling this?  If it is generic logic on the API side it might be difficult to introduce a different method name.

Reply:
@skateman Any idea?

Reply:
Yeah, that method name is probably not okay, cc @martinpovolny @abellotti

Reply:
@skateman Do you know where API is calling this? Can you post the link here?

Reply:
I don't think that the API would be able call this, but if the method is named correctly, we can work with it. Others might know more ...

Reply:
@skateman Sorry did not make it clear. Regardless of the method name, where would the new method be called in API?

Reply:
Can we name it `update_properties`?